scheduler/src/cook/mesos/util.clj

```diff
@@ -367,9 +361,11 @@
                               (get-completed-jobs-by-user db user start end limit state name-filter-fn)
                               (get-active-jobs-by-user-and-state db user start end state name-filter-fn)))
         jobs-by-state (mapcat get-jobs-by-state states)]
-    (->> jobs-by-state
-         (sort-by :job/submit-time)
-         (take limit))))
+    (timers/time!
+      (timers/timer ["cook-mesos" "scheduler" "get-jobs-by-user-and-states-duration"])
```

Comment:
What value does this timer add over the ones in `get-active-jobs-by-user-and-state` and `get-completed-jobs-by-user`?

Reply:
It decomposes the time spent in the outer timer into this particular component --- in particular, instance state lookup.